io/include/pcl/io/hdl_grabber.h

```diff
@@ -270,22 +275,22 @@ namespace pcl
       fireCurrentSweep ();
 
       void
-      fireCurrentScan (const unsigned short startAngle,
-                       const unsigned short endAngle);
+      fireCurrentScan (const uint16_t startAngle,
+                       const uint16_t endAngle);
       void
       computeXYZI (pcl::PointXYZI& pointXYZI,
-                   int azimuth,
+                   uint32_t azimuth,
```

Comment:
In line 263 `last_azimuth_` is only 16 bits. Which one should we adopt?

Reply:
Yes, It seems correct to adopt 16 bits.